spec/models/customer_spec.rb

```diff
@@ -69,5 +69,9 @@
     it 'can count successful transactions' do
       expect(@customer_1.successful_transactions_count).to eq(1)
     end
+
+    it "can return its full name" do
```

Comment:
Thank you! I think we should talk to the teachers if this should live in the model or view (or even if we want to add it as a column and create a parameter in our controller) but this keeps us moving for now.